fe/fe-core/src/main/java/com/starrocks/connector/iceberg/io/IcebergCachingFileIO.java

```diff
@@ -132,7 +133,12 @@ public void initialize(Map<String, String> properties) {
             switch (type.toLowerCase(Locale.ROOT)) {
                 case "hive":
                 case "rest":
-                    wrappedIO = new HadoopFileIO(conf);
+                    if (properties.getOrDefault(KEY_ENABLE_TABULAR_SUPPORT, "false").equals("true")) {
```

Comment:
It's better to optimize this block for legibility

Reply:
is there any good advice?